src/D8/EckEntityTrait.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+namespace DrevOps\BehatSteps\D8;
+
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\Gherkin\Node\TableNode;
+use Drupal\user\Entity\User;
+
+/**
+ * Trait ContentEntityTrait.
+ *
+ * @package IntegratedExperts\BehatSteps\D8
+ */
+trait EckEntityTrait {
+
+  /**
+   * Custom eck content entities organised by entity type.
+   *
+   * @var array
+   */
+  protected $eckEntities = [];
+
+  /**
+   * Create eck entities.
+   *
+   * Provide entity data in the following format:
+   * | title  | field_marine_animal     | field_fish_type | ... |
+   * | Snook  | Fish                    | Marine fish     | 10  |
+   * | ...    | ...                     | ...         | ...     |
+   *
+   * @Given :bundle :entity_type entities:
+   */
+  public function eckEntitiesCreate($bundle, $entity_type, TableNode $table) {
```

Comment:
please update all method names to start with `eck` as it is used as a namespace.